jfrog_platform_cloud/assets/logs/jfrog_artifactory.yaml

```diff
@@ -27,96 +22,160 @@ facets:
     name: Status Code
     path: http.status_code
     source: log
-  - groups:
-      - Web Access
-    name: URL Path
-    path: http.url
-    source: log
   - groups:
       - Web Access
     name: User-Agent
     path: http.useragent
     source: log
-  - groups:
-      - Source Code
-    name: Thread Name
-    path: logger.thread_name
-    source: log
   - groups:
       - Web Access
     name: Client IP
     path: network.client.ip
     source: log
-  - groups:
-      - User
-    name: User ID
-    path: usr.id
+  - facetType: range
+    groups:
+      - Measure
+    name: Duration
+    path: duration
     source: log
+    type: double
+    unit:
+      family: time
+      name: nanosecond
   - groups:
-      - JFrog Cloud
-    name: Tenant ID
-    path: jfrog.cloud.tenant_id
+      - Event
+    name: Event Name
+    path: evt.name
     source: log
-  - groups:
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Data Changed
-    path: jfrog.cloud.data_changed
+    name: Access repository path
+    path: repository_path
```

Comment:
are you sure this facet is being used ? I see there is already remapper from `repository_path` to `jfrog.cloud.repo_path`

Reply:
This is a mistake, I'm adding a new facet for that and removing this one. Thanks!